docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A fresh approach for date and time inputs
+description: Check out what's new with the next-gen pickers in earlier v6 versions.
+date: 2022-12-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+---
+
+There's a lot of exciting news in MUI X v6, but there's hardly anything comparable to the revamp we're delivering for the [Date and Time Pickers](https://next.mui.com/x/react-date-pickers/getting-started/).
+
+It's the result of our research and effort to improve these inputs' usability and overall customizability.
+We kept the best aspects of the previous versions, and we're fleshing out the package with several improvements and one fundamental change: a new input type to replace the masked text fields.
+
+## The new date and time fields
+
+This new input type we're talking about is a set of components shortly named [Fields](https://next.mui.com/x/react-date-pickers/fields/).
+They are the base for the new pickers and a revolution in terms of keyboard usability.
+
+In contrast with the masked inputs that only display simple text behind a formatting mask, and were often cumbersome, especially when editing dates, the new fields are smart and specialized in editing date and time numbers.
```

Comment:
This long run-on sentence is best split up in two.

```suggestion
The earlier masked inputs were often quite cumbersome to use—especially when editing dates.
The new Fields, by contrast, are optimized for the best possible editing experience.
```

Reply:
Shouldn't we use `Masked inputs` + `Fields` or `masked inputs` + `fields` ?

Reply:
@flaviendelangle which of these does "masked inputs" refer to? It's not clear to me in context, but this would be the difference in terms of style:
a. MUI component = Masked Inputs
b. general concept = masked inputs

Reply:
If that's not clear for you, then we probably need to rephrase.

The input of the old pickers was based on a mask library called [rifm](https://www.npmjs.com/package/rifm). It was not usable as a standalone component so its name was not really normalized (the private component is called `KeyboardDateInput`)

The input of the new pickers are based on the Field components, which are usable as standalone components.

But from a user perspective, `mask` is a term that people only encountered if they tried to customize the behaviors. It's not a word we are referring a lot to in the doc like `field` will be.

Maybe we could rephrase to something like: 

```suggestion
In contrast with the previous generation of pickers, whose input editing was based on a [mask approach](https://www.npmjs.com/package/rifm) and were often cumbersome, the new fields are smart and specialized in editing date and time values.
```

I am replacing `numbers` by `values` (if you have a better idea) because we support letter sections like full letter months (`January`) as well.

And to be honest I don't understand the `especially when editing dates`, what else are they doing than editing dates ? :angel:

Reply:
> And to be honest I don't understand the especially when editing dates, what else are they doing than editing dates ?

Just a quick note on this one: editing vs. inserting

Reply:
Oh OK, in that case maybe "editing" is not the right word because we are also using it to describe the whole process (the initial filling and the modification of already existing values).

Maybe `modifying a value` would describe it better.